### ISSUE TITLE
Feature request: allow user to control order of parameters in tracking UI

### ISSUE THREAD

**Issue by gsganden** [mlflow/mlflow#105]
### Describe the problem
There are often natural ways to order parameters, e.g. to group related quantities. Currently the UI orders them alphabetically. It would be nice to be able to control the ordering, e.g. to match the order in which the logging happens or by providing a list in the desired order.

**Comment by mateiz** [comment]
Thanks, that makes sense. We want to eventually let you save views or filters of the experiment page to make this easier. One piece to do for that is figure out whether there's a good table control that already supports dragging/dropping columns, etc that we can use. I was curious, what type of workload do you have where there are a lot of parameters and how would you like to order them?

**Comment by gsganden** [comment]
I'm doing deep learning for image classification. I would to be able to group together parameters indicating whether or not various data augmentations were used (e.g. flipping each image, perturbing its brightness, etc.), for instance, separately from other hyperparameters such as the learning rate.

> We want to eventually let you save views or filters of the experiment page to make this easier. 

I like that approach.

**Comment by mateiz** [comment]
OK, makes sense. For now, I'd suggest trying to use a hierarchical structure in your key names (e.g. aug.flipp, aug.brightness, etc) to do this, but eventually we'll let you configure the list of things shown and save filters as I mentioned.

**Comment by mateiz** [comment]
FYI, pull request #186 lets you specify an order for parameters now, but we still need to make the filter settings for an experiment saveable (that will be a different pull request).

**Comment by lorenzwalthert** [comment]
Can we for now simply allow the user to select alphabetical order or input order? E.g. in my `MLproject` file, I already put the parameter in an order I like. That would probably be order of magnitudes simpler than an external configuration mapping.

**Comment by chenmoneygithub** [comment]
Close inactive issue.
